### DISCUSSION TITLE
Error while deploying prebuilt artifacts on vercel using the vercel CLI

### DISCUSSION THREAD

**Discussion by jonathandsouza** [vercel/vercel#8564] [Help] [answered]
Hi There, 

I have been trying to deploy a nextJS starter template using the Vercel CLI following the instruction below.

https://vercel.com/guides/using-vercel-cli-for-custom-workflows 

![image](https://user-images.githubusercontent.com/6910063/190227007-4a9a4064-5ef6-4325-b796-562b3e3dbc39.png)

I am getting the following error when I try to deploy the artifact. 

![image](https://user-images.githubusercontent.com/6910063/190227158-135e60c2-64e0-4aa1-8032-e950022e5579.png)

Any guidance or help would be much appreciated.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @jonathandsouza! Are you able to share you test repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? It looks like there is a problem with the build, but I can't tell exactly what from this error alone.

**Reply by jonathandsouza** [reply]
HI @amyegan, 

Thank you for taking the time to reply to me. I am incredibly humbled. 

Here is the test repo, It's a stripped-down version of a standard creat-next-app. 

https://github.com/jonathandsouza/vercel-prebuilt-deployment-test

Environment:
Windows 11
node v16.16.0
Vercel CLI 28.2.5

Deployment link: https://vercel.com/jdsouza-freetimecomp/vercel-prebuilt-deployment-test/CaFVEimmSVghu52LhzEUxbPPrpkR

The build using "vercel build" goes through perfectly.

![image](https://user-images.githubusercontent.com/6910063/190252532-082bec63-68ed-4913-a1ac-45e177814f37.png)

The problem occurs when I try to run "vercel deploy --prebuilt"

![image](https://user-images.githubusercontent.com/6910063/190252830-f2e37e40-ea20-4999-8301-e849dba9947d.png)

**Reply by amyegan** [reply]
Thanks for sharing this! I was able to build and deploy it without error - https://vercel-prebuilt-deployment-test-dfpczs9by-amyegan.vercel.app/

```
Retrieving list of deployment files...
Downloading 17 deployment files...
Using prebuilt build artifacts...
Generated build outputs:
- Static files: 16
- Serverless Functions: 0
- Edge Functions: 0
Deployed outputs in 1s
Done with "."
```

I'm working from a Mac, so my guess is that the problem is related to Windows. I will ask the Build engineers to take a look. 

Have you tried with any older versions of the CLI? I wonder if this error is being caused by a recent change.

**Reply by amyegan** [reply]
The Engineering team is already working on it 😄 https://github.com/vercel/vercel/pull/8530
